speeduino/comms_CAN.cpp

```diff
@@ -36,7 +36,17 @@ void sendVAGCluster()
   DashMessage(CAN_VAG_VSS);
   Can0.write(outMsg);
 }
-
+void reciveCANwbo()
+  {
+    if (inMsg.id == 400 && configPage15.canREWBOCAN == true) // RusEFI CAN Wideband support
```

Comment:
I'd love if you could throw a link to my repo in here (I'm the creator): https://github.com/mck1117/wideband